tasmota/xdrv_12_home_assistant.ino

```diff
@@ -346,7 +348,12 @@ void NewHAssDiscovery(void)
               TasmotaGlobal.version, TasmotaGlobal.mqtt_topic, SettingsText(SET_MQTT_FULLTOPIC), PSTR(SUB_PREFIX), PSTR(PUB_PREFIX), PSTR(PUB_PREFIX2), Hass.RelLst, switch_mode, switch_name,
               stemp5, Settings->flag.mqtt_response, Settings->flag.button_swap, Settings->flag.button_single, Settings->flag.decimal_text, Settings->flag.not_power_linked,
               Settings->flag.hass_light, Settings->flag3.pwm_multi_channels, Settings->flag3.mqtt_buttons, Settings->flag4.alexa_ct_range, Settings->flag5.mqtt_switches,
-              Settings->flag5.fade_fixed_duration, light_controller.isCTRGBLinked(), Light.subtype, stemp6);
+              Settings->flag5.fade_fixed_duration, 
+#ifdef USE_LIGHT
+              light_controller.isCTRGBLinked(), Light.subtype, 
+#endif // USE_LIGHT
```

Comment:
This breakes the discovey. Parameters ofHASS_DISCOVER_DEVICE got scrambled.

Reply:
Oh whoops. Didn't notice it, what would be reasonable default values for this?

Reply:
Well, it is required from HATasmota module of Home Assistant, so data must be there. May be forcing to be relay?